### ISSUE TITLE
CheckBox becomes invisiable when a lot of text is placed as a message in SimpleCheckDialog.

### ISSUE THREAD

**Issue by isabsent** [eltos/SimpleDialogFragments#65]
```
        SimpleCheckDialog.build()
                .msg(R.string.dialog_message_eula)
                .pos(android.R.string.ok)
                .neg(R.string.btn_decline)
                .check(false)
                .checkRequired(true)
                .label(R.string.label_eula)
                .cancelable(false)
                .show(this, HANDLE_EULA);
```
The result of avove for `com.github.eltos:SimpleDialogFragments:3.2` is:

![Screenshot_2020-11-29-09-48-53-727](https://user-images.githubusercontent.com/4918293/100535307-c0396680-3228-11eb-9b24-d8b11dbb8250.jpg)

**Comment by isabsent** [comment]
The bug is in `SimpleDialogFragments/simpledialogfragment/src/main/res/layout/simpledialogfragment_custom_view.xml`

It should be as:

```
<?xml version="1.0" encoding="utf-8"?>
<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
    android:orientation="vertical"
    android:layout_width="match_parent"
    android:layout_height="wrap_content">

    <ScrollView
        android:layout_width="match_parent"
        android:layout_height="0dp"
        android:layout_weight="1"
        android:requiresFadingEdge="vertical"
        android:clipToPadding="false">

        <LinearLayout
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:orientation="vertical">

            <View
                android:id="@+id/textSpacerNoTitle"
                android:visibility="gone"
                android:layout_width="match_parent"
                android:layout_height="8dp" />

            <TextView
                android:id="@+id/customMessage"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:layout_marginTop="8dp"
                android:paddingEnd="?attr/dialogPreferredPadding"
                android:paddingStart="?attr/dialogPreferredPadding"
                style="@style/TextAppearance.AppCompat.Subhead" />

        </LinearLayout>
    </ScrollView>

    <FrameLayout
        android:id="@+id/customView"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"/>

</LinearLayout>
```

The difference is in the place of 
```
        android:layout_height="0dp"
        android:layout_weight="1"
```

**Comment by eltos** [comment]
Thanks @isabsent for the bug report. I fixed it with v3.3.1 that I just released.

PS: you might want to use `.msgHtml()` for such long texts. Check out the [testApp](https://github.com/eltos/SimpleDialogFragments/blob/91151fd46a689bc7b0945d381724720b3d5f59af/testApp/src/main/java/eltos/simpledialogfragments/MainActivity.java#L160)

**Comment by isabsent** [comment]
> you might want to use `.msgHtml()`
Yes, already did it.
